app/main.py

```diff
@@ -1,13 +1,50 @@
 def format_linter_error(error: dict) -> dict:
-    # write your code here
-    pass
+
+    return {
+        "line": error["line_number"],
+        "column": error["column_number"],
+        "message": error["text"],
+        "name": error["code"],
+        "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # write your code here
-    pass
+
+    return {
+        "errors": [
+            {
+                "line": error["line_number"],
+                "column": error["column_number"],
+                "message": error["text"],
+                "name": error["code"],
+                "source": "flake8"
+            } for error in errors
+        ],
+        "path": file_path,
+        "status": "failed"
```

Comment:
![image](https://github.com/mate-academy/py-linter-formatter/assets/103201010/ac4f332e-f435-4e1e-b48c-234100d0709e)
fix this according to the task